update vale and fix ini (#56)
ConsenSys/doc.goquorum: docs/Reference/Consensus/Raft-RPC-API.md

## `raft_addLearner`

> API to add a new node to the network as a learner node. The learner node syncs with network and can transact but will not be part of Raft GoQuorum and hence will not provide block confirmation to minter node.

Reviewer: ```suggestion
API to add a new node to the network as a learner node. The learner node syncs with network and can transact but is not part of Raft GoQuorum and does not provide block confirmation to minter node.
```
